impeller/renderer/backend/vulkan/swapchain/ahb/ahb_texture_pool_vk.cc

```diff
@@ -10,12 +10,8 @@ namespace impeller {
 
 AHBTexturePoolVK::AHBTexturePoolVK(std::weak_ptr<Context> context,
                                    android::HardwareBufferDescriptor desc,
-                                   size_t max_entries,
-                                   std::chrono::milliseconds max_extry_age)
-    : context_(std::move(context)),
-      desc_(desc),
-      max_entries_(max_entries),
-      max_extry_age_(max_extry_age) {
+                                   size_t max_entries)
```

Comment:
I think two is too small of a number, or we should potentially let it float above max_entries without GC. We can handle that as a follow up.